docs/samples/contact-center/app.js

```diff
@@ -361,3 +375,54 @@ function expandAll() {
   });
 }
 
+function holdCall() {
+  holdElm.disabled = true;
+  task.hold(taskId).then(() => {
+    console.info('Call held successfully');
+    resumeElm.disabled = false;
+  }).catch((error) => {
+    console.error('Failed to hold the call', error);
+    holdElm.disabled = false;
+  });
+}
+
+function resumeCall() {
+  resumeElm.disabled = true;
+  task.resume(taskId).then(() => {
+    console.info('Call resumed successfully');
+    holdElm.disabled = false;
+  }).catch((error) => {
+    console.error('Failed to resume the call', error);
+    resumeElm.disabled = false;
+  });
+}
+
+function endCall() {
+  endElm.disabled = true;
+  task.end(taskId).then(() => {
+    console.log('Call ended successfully');
+    holdElm.disabled = true;
+    resumeElm.disabled = true;
+    wrapupElm.disabled = false;
+    wrapupCodesDropdownElm.disabled = false;
+  }).catch((error) => {
+    console.error('Failed to end the call', error);
+    endElm.disabled = false;
+  });
+}
+
+function wrapupCall() {
```

Comment:
If the call is not ended from our kitchen sink app, then does the wrapup button become active and are we able to wrapup?

Reply:
Unless the call ends, we don't enable this button (in the kitchen sink).